onnx/cpp2py_export.cc

```diff
@@ -372,6 +372,12 @@ PYBIND11_MODULE(onnx_cpp2py_export, onnx_cpp2py_export) {
     checker::check_node(proto, ctx, lex_ctx);
   });
 
+  checker.def("check_function", [](const py::bytes& bytes, const checker::CheckerContext& ctx) -> void {
```

Comment:
Is it possible to add a link in the documentation (here: https://github.com/onnx/onnx/blob/main/docs/docsgen/source/api/checker.rst) so that this function appears on the documentation?

Reply:
Good idea! I will try to add it in this PR.